product_input_interface/allowed_products.py

```diff
@@ -0,0 +1,28 @@
+class WarehouseProducts:
+    __ALLOWED_PRODUCTS_TYPE = ["helmet", "rifle", "medkit"]
+    __MAX_PRODUCT_NAME_LENGTH = 60
+    __MIN_PRODUCT_NAME_LENGTH = 1
+
+    def print_allowed_product_types(self):
+        print("Allowed product types:")
+        for item in self.__ALLOWED_PRODUCTS_TYPE:
+            print(f"- {item}")
```

Comment:
we have some encapsulated part of the code `WarehouseProducts` - usually in such "deep" parts of the code we should avoid side-effects like using `print` function because such code is harder to test and maintain. I would suggest to return here allowed product types and someone who is invoking that function can decided what to do with that (print or send via internet or send via email etc.)
Eventually we can write `get_allowed_product_types_formatted` and return allowed product types formatted with `- {item}` as string